app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,92 @@
+class OrdersController < ApplicationController
+  before_action :logged_in_user, :load_data_order
+
+  def new; end
+
+  def create
+    error_messages = handle_data_product_and_quantity_for_cart
+    if check_error_messages? error_messages
+      @error_messages = error_messages
+      render :new
+      return
```

Comment:
```suggestion
      return render :new
```
thử ri luôn dc ko cu

Reply:
Dạ oke rồi chị